.travis.yml

```diff
@@ -3,8 +3,12 @@ language: python
 python:
   - "3.5"
   - "3.6"
+  - "3.7-dev"
 install:
-  - pip install tox
+  - pip install -U setuptools
+  - pip install -U pip
+  - pip install -U wheel
+  - pip install -U tox
 script:
   - export TOXENV=py`python -c 'import sys; print("".join(map(str, sys.version_info[:2])))'`
```

Comment:
Also just `TOXENV=py` should work.

Reply:
I have no idea how it should work, cuz we need explicit env namig

Reply:
Why? You don't. It will pick up whatever current `python` points to by default, but will allow `-e py37`-like stuff if you need.